CometServer/Services/Resolve/ResolveService.cs

```diff
@@ -327,29 +331,38 @@ internal void ResolvePartitionFromDataStore(NpgsqlTransaction transaction, strin
         /// <returns>
         /// The <see cref="string"/>.
         /// </returns>
-        private string ResolvePartition(string partition, string typeName)
+        private IList<string> ResolvePartition(string partition, string typeName)
```

Comment:
my preference would be to return an IEnumerable instead of IList, if you agree please change, if not, leave it as is and I go with your opinion.